lib/cc/services/github_pull_requests.rb

```diff
@@ -17,8 +17,12 @@ class Config < CC::Service::Config
   self.description = "Update pull requests on GitHub"
 
   BASE_URL = "https://api.github.com"
-  BODY_REGEX = %r{<b>Code Climate</b> has <a href=".*">analyzed this pull request</a>}
-  COMMENT_BODY = '<img src="https://codeclimate.com/favicon.png" width="20" height="20" />&nbsp;<b>Code Climate</b> has <a href="%s">analyzed this pull request</a>.'
+
+  DETAILED_COMMENT_BODY = '<img src="https://codeclimate.com/favicon.png" width="20" height="20" />&nbsp;<b>Code Climate</b> found <a href="%s">%s and %s</a>.'
+  DETAILED_COMMENT_REGEX = %r{<b>Code Climate</b> found <a href=".*">.*issues.*</a>}
```

Comment:
Looks like this regex can fail in the case of 1 fixed issue. I think the "s" needs to be optional